sgl-kernel/python/sgl_kernel/moe.py

```diff
@@ -1,5 +1,5 @@
 from typing import Optional
-
```

Comment:
Can we separate the changes to sgl-kernel into a separate PR, because I need to release a new version of sgl-kernel.